app/models/global_issue_template.rb

```diff
@@ -22,8 +22,17 @@ class GlobalIssueTemplate < ActiveRecord::Base
   attr_accessible :title, :tracker_id, :issue_title, :description, :note,
                   :enabled, :project_ids, :position, :author
 
+  scope :enabled, lambda { where(enabled: true) }
```

Comment:
__[rubocop]__
* Use the `-> { ... }` lambda literal syntax for single line lambdas. [:ref](https://github.com/bbatsov/ruby-style-guide#lambda-multi-line)